Assets/Scripts/Dialogue System/RunTime/DialogueNodeData.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[Serializable]
+public class DialogueNodeData
+{
+    public string NodeGuID;
+    public string DialogueText;
+    public Vector2 position;
```

Comment:
The base data storage for this and NodeLinkData is set up in a less than ideal way. Ideally, nodes should be able to access their children/parents through references, rather than searching a list by GUID.

Reply:
not changing now